z3c/dependencychecker/dependencychecker.py

```diff
@@ -501,9 +514,9 @@ def imports_from_doctests(path):
     for path, dirs, files in os.walk(path):
         for filename in [os.path.abspath(os.path.join(path, filename))
                          for filename in files
-                         if fnmatch.fnmatch(filename, '*.txt')
-                         or fnmatch.fnmatch(filename, '*.rst')
-                         or fnmatch.fnmatch(filename, '*.py')]:
+                         if fnmatch.fnmatch(filename, '*.txt') or
+                         fnmatch.fnmatch(filename, '*.rst') or
```

Comment:
You're sure? If I read https://hg.python.org/peps/rev/3857909d7956, it seems like you should break before the binary operator. If the break is before the "or/and", then the "or/and" is the first one on the line, right?